src/tests/JIT/HardwareIntrinsics/Arm/Shared/_SveImmBinaryOpTestTemplate.template

```diff
@@ -150,7 +150,7 @@ namespace JIT.HardwareIntrinsics.Arm
 
             public void RunStructFldScenario({TemplateName}BinaryOpTest__{TestName} testClass)
             {
-                var result = {Isa}.{Method}(_fld1, _fld2, {Imm});
+                var result = {Isa}.{Method}(_fld1, _fld2, Imm);
```

Comment:
This is missing a test where we pass `invalidImm` as input. See `test.RunBasicScenario_UnsafeRead_InvalidImm` in `_SveImmUnaryOpTestTemplate.template` for example.

Reply:
Got it. I noticed the `InvalidImm` tests in other templates look like this:
```csharp
bool succeeded = false;
try
{
    var result = {Isa}.{Method}(
        Unsafe.Read<{Op1VectorType}<{Op1BaseType}>>(_dataTable.inArrayPtr),
        {InvalidImm}
    );
}
catch (ArgumentOutOfRangeException)
{
    succeeded = true;
}
```

We aren't doing anything with `succeeded`. Is this intentional?

Reply:
> We aren't doing anything with `succeeded`. Is this intentional?

Unfortunately, it is not. It is missing this code:

```c#
            if (!succeeded)
            {
                Succeeded = false;
            }
```

Opened https://github.com/dotnet/runtime/issues/104809 to track it. Feel free to just update this one and for others we will take care of as part of fixing the overall issue.